### ISSUE TITLE
Size & Position panel: Add x & y position input fields

### ISSUE THREAD

**Issue by o-fernandez** [GoogleForCreators/web-stories-wp#3636]
**[Figma Designs](https://www.figma.com/file/bMhG3KyrJF8vIAODgmbeqT/Sam---Visual-Design-Frames?node-id=221%3A13978)**

A user should be able to specify x & y coordinates for any element on the canvas. This doesn't apply to the page itself, but media, text, and shape elements should all have this in the design panel.

## QA Instructions

***Note that this is temporary work. A new design is coming with proper definitions and layout and this work thus doesn't need to be that rigorously validated.***

1. Add any type of element
1. Observe that in the design panel there are inputs for setting X and Y position of the element
1. Use either numbers or keyboard arrows to manipulate element position and observe that the position only updates on blur
1. Also observe that limits for position allow the element to be placed just outside the canvas touching the border on all sides both when entering number using number keys or stepping using arrow keys (the former is validated on blur, the latter directly on input)
a. This latter restriction should also work with rotated elements, but there might be some weird edge cases here

**Comment by csossi** [comment]
Verified in QA
